### PR TITLE
fix: send erc20

### PR DESCRIPTION
**Pull request by alaibe**

### CODE REVIEW

File: services/wallet/bridge/transfer.go [hunk 1]
```diff
@@ -34,9 +41,45 @@ func (s *TransferBridge) CalculateFees(from, to *params.Network, token *token.To
 
 func (s *TransferBridge) EstimateGas(fromNetwork *params.Network, toNetwork *params.Network, from common.Address, to common.Address, token *token.Token, amountIn *big.Int) (uint64, error) {
 	// TODO fix for ERC20
```
Comment (J-Son89) [inline services/wallet/bridge/transfer.go]: is the TODO needed still?

Comment (alaibe) [reply services/wallet/bridge/transfer.go]: nope, fixed

**Review by dlipicar — APPROVED** [review]
Nice! I assume the estimation will be different for ERC721, we could add that one in the future

**Comment by alaibe** [comment]
@dlipicar we already have it for erc721!

**Pull request merged**